DigitalLearningSolutions.Data/DataServices/CentresDataService.cs

```diff
@@ -236,5 +228,29 @@ FROM Centres
             var ipPrefixes = ipPrefixString?.Split(',', StringSplitOptions.RemoveEmptyEntries);
             return ipPrefixes ?? new string[0];
         }
+
+        public IEnumerable<CentreRank> GetCentreRanks(DateTime dateSince, int regionId)
+        {
+            return connection.Query<CentreRank>(
+                @"SELECT 
+                        RANK() OVER (ORDER BY tc.CentreIDCount DESC) AS [Rank],

```

Comment:
Do we need the "[]" around Rank & Sum?

Reply:
They're SQL keywords, sometimes using keywords like this does cause error so I just left them in the brackets (as per the old SP). Renaming all of these to something that isn't a keyword would resolve this anyway.